### ISSUE TITLE
Review System

### ISSUE THREAD

**Issue by rizwank** [neinteractiveliterature/intercode#1466]
I get that this is a far bigger policy question than it is a technology question, but I'd love to be able to review games afterwards - it would help others understand (especially newbies) GM styles between writers, and reviews of a game that has run before. Even after a decade of Intercon I don't know most of the GMs and it's a crapshoot on whether a game with a great blurb has been considered a great game in the past.

I can see all sorts of reasons why this is a bad idea, so I'll understand if this is closed as a WONT FIX.

**Comment by nbudin** [comment]
Oh man, that is a _really_ interesting idea.

I think you're right that it's a policy question, and so I'm going to throw this to @dkapell as far as Intercon goes.  If we did implement this, it would be an optional feature that cons could enable/disable as they saw fit, I think, since I'm sure cons would have different opinions about whether they want it.

**Comment by diewald** [comment]
Feedback for games has been collected for a long time, in post-con surveys. I don’t know if and when we stopped collecting it. I know a couple of things:

I used to have access to it as a member of the Bid Committee - where it proved very useful in evaluating bids. I have not had access to this information for several conventions now. I would like to have that access again.
I don’t remember ever getting any feedback about the games I’ve run. Given my recent memory issues, it’s possible that I’ve forgotten it, or that it’s buried in my unending inbox. I would like to know what people thought of my games, good and bad. 
One set of reviews is never sufficient. Reviews are dependent on the set of players involved, their decisions, the phase of the moon, and other crazy things. I’ve seen good games take really strange turns into not-so-good, despite the same game, the very same text, working amazingly well with a different set of players. 

Consider 1 & 2 as two potential use cases, and 3 as something to consider for a more widely visible use case.

Jeff

[Migraine at 21 months. Waiting on test results. Typos, missing words, and other language oddities may result.]

**Comment by dkapell** [comment]
We collected it last year, and we will likely collect it next year.  It's an increasingly large amount of work to generate the survey, so we need to revisit this.  
It's been a long time since every member of the proposal committee had access, I don't remember when that was, and looking at the survey history, it predates K.  We've historically been really bad about passing on that feedback, because the time for that would be a few months after the con, when the concom from the previous year has dispersed some.

I like the idea of building a survey feature into the website, or as a standalone site that uses the oauth features to pull your games, and maybe i'll take a swing at that soon.

**Comment by diewald** [comment]
I had access to the Journey survey results for every year Tim put out the survey as registrar, because I specifically asked for it. I don't think anyone else on the Bid Committee was looking at them. O was the last year there were any significant results. (@dkapell - you have access to the M, the failed N, and the O surveys there. You may also be able to see the L survey.)

To handle case 3, it would be nice to aggregate game survey results across cons. **Question:** Does it matter who runs the game for aggregation? _The Tales of Irnh_ have been run by at least 7? different GMs across the years at Intercon.

**Comment by dkapell** [comment]
I see 82 responses to teh post O survey, 81 for P, 97 for Q, and R used a different system, but we got a bunch of results.

I think using surveing software, and then manually distributing the data is doomed to failure, but Intercode 2's oauth would let a lot of that happen automatically, albeit with supervision.  I'll work on a design doc for this.

**Comment by rizwank** [comment]
I’d suggest leveraging this growing platform to collect it - would probably get more compliance. 

I can see some downsides / GMs that have one bad session could get a “rap”, one off games that aren’t repeated don’t have game level reviews / calibration with a small data set, etc. might be a good pre-con session to discuss ideas.  

I think the current blind system favors those closely embedded in the community and who are NE residents who know everyone. I’ve been attending since J and still scratch my head if it isn’t once of a few groups I know <1>. I personally vet my choices through friends, which feels awkward. I’d suggest that makes the con a little less appealing for outsiders to join and bring their pizzaz. Whether that’s ok goes to the ethos of the con, which isn’t up to me. 

What I do know is that IC is one of the most welcoming events I’ve ever been to, and this may help. 

+1 to the tag system btw. It’d be awesome to get a style readout, even if imprecise. 

<1> I just realized this also disadvantages newish GMs who are 1-2 cons in but don’t have the repute (yet) of, say, AEgames. It may help spread out the land rush at signup day. 

Appreciate the conversation, will gladly support whenever you end up. 

Rk

**Comment by diewald** [comment]
@dkapell I agree, and look forward to seeing what the design looks like. I didn't have access to the P, Q, or R data, which would have helped my (and others') Bid Committee work.

@rizwank My daughter Julie has been interrogating me about several of the games - and it turns out that I didn't know a lot of the GMs bidding games this year. This is a very good thing, encouraging new people to bid.

**Comment by nbudin** [comment]
Hey folks!  @dkapell is right to separate out these concepts, and this ticket is specifically about the second of them (a public-facing review system).

We actually already have a separate ticket for the first: #7.  This is a useful discussion to have, but I think it's better had over there, so I'm going to ask that you move it to that ticket.  Thanks!

**Comment by rizwank** [comment]
Bumping this for the new year if it’s still appropriate.

**Comment by nbudin** [comment]
Closing based on the conversation in #7.  Would love to see the post-event survey app happen, but it's a non-Intercode thing so should be tracked elsewhere.
